scripts/utils/import-metrics.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Script using mimirtool for prometheus tsdb collection
+#
+# Usage: ./import-metrics.sh path/to/kubeconfig.yaml
+#
+# Important: Must set inputs for selector and time range
+# 
+# For all metrics use selector='{__name__!=""}'
+# Time format YYYY-MM-DDThh:mm:ssZ  ex: "2025-01-31T00:00:00Z"
+# bash helpers: 
+#      date -u +"%Y-%m-%dT%H:%M:%SZ"
+#      date -u -v-3H +"%Y-%m-%dT%H:%M:%SZ"
+#      [-v[+|-]val[y|m|w|d|H|M|S]]
+selector='{__name__!=""}' 
+from="$(date -u -v-2H +"%Y-%m-%dT%H:%M:%SZ")"
+to="$(date -u -v-0H +"%Y-%m-%dT%H:%M:%SZ")"
+
+# set kubeconfig
+export KUBECONFIG=$1
+
+#TODO: confirm access to cluster
```

Comment:
You could run a non-modifying `kubectl` command which should return a success or error exit code that you can conditionally exit the script on. Even something like `kubectl get all -A || exit 1` would probably be fine.